tests/testAutopas/tests/containers/LinkedCellsVersusVerletClusterListsTest.cpp

```diff
@@ -5,12 +5,13 @@
  */
 
 #include "LinkedCellsVersusVerletClusterListsTest.h"
+#include "autopas/containers/verletClusterLists/traversals/VerletClustersTraversal.h"
 
-LinkedCellsVersusVerletClusterListsTest::LinkedCellsVersusVerletClusterListsTest()
-    : _verletLists(getBoxMin(), getBoxMax(), getCutoff(), 0.1 * getCutoff(), 2),
-      _linkedCells(getBoxMin(), getBoxMax(), getCutoff(), 0.1 * getCutoff(), 1. /*cell size factor*/) {}
-
+template <autopas::DataLayoutOption dataLayout, bool useNewton3>
 void LinkedCellsVersusVerletClusterListsTest::test(unsigned long numMolecules, double rel_err_tolerance) {
+  Verlet _verletLists{getBoxMin(), getBoxMax(), getCutoff(), 0.1 * getCutoff(), 2};
```

Comment:
last one is number of particles per cluster correct? Why 2?

Reply:
Correct. I think I just took over the parameters from before. I will remove the "2" to let it use the default of "4".